apps/s_server.c

```diff
@@ -149,7 +149,7 @@ static int dtlslisten = 0;
 static int early_data = 0;
 
 #ifndef OPENSSL_NO_PSK
-static const char psk_identity[] = "Client_identity";
+char *psk_identity = "Client_identity";
```

Comment:
There's a travis failure due to you removing the 'static'. Mind updating? Otherwise, this looks good to me.